enigma-cli/src/main/java/cuchaz/enigma/command/Main.java

```diff
@@ -39,8 +39,10 @@ public static void main(String... args) {
 		} catch (CommandHelpException ex) {
 			Logger.error(ex);
 			logEnigmaInfo();
-			Logger.info("Command {} has encountered an error! Usage:", ex.command.name);
-			printHelp(ex.command);
+			Logger.info("Command {} has encountered an error! Usage:", ex.command.getName());
+			StringBuilder help = new StringBuilder();
+			appendHelp(ex.command, help);
+			Logger.info(help.toString());
```

Comment:
This is more user-facing, so I'm not sure if we should be even using a logger here

Reply:
I'll address that later!